USFMConverter/Core/Render/RenderHTML.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using USFMConverter.Core.ConstantValue;
+using USFMConverter.Core.Data;
+using USFMToolsSharp.Models.Markers;
+using USFMToolsSharp.Renderers.HTML;
+
+namespace USFMConverter.Core.Render
+{
+    public class RenderHTML : RenderDocument
```

Comment:
This class is confusingly named.  It should be called something like "HTMLDocument."